ui/src/app/app-routing.module.ts

```diff
@@ -42,7 +43,7 @@ const routes: Routes = [
     path: '',
     component: SignedInComponent,
     canActivate: [SignInGuard],
-    canActivateChild: [SignInGuard, RegistrationGuard],
+    canActivateChild: [SignInGuard, RegistrationGuard, EraCommonGuard],
```

Comment:
We need a plan for how this interacts with `RegistrationGuard` and the unregistered page. As it stands in this PR, you'd need to be manually ID verified before you could get to the dashboard to click the ERA commons link, which defeats the purpose of using ERA commons for ID verification. At the same time, we probably still want a manual IDV step so we can continue whitelisting initial users - I don't know whether this has been discussed yet.

Previous treatment for an unregistered user:
- Sign in
- RegistrationGuard redirects to /unregistered
- unregistered page autosubmits tos/training/manual IDV request/"demographics survey"

Possible new treatment?
- Sign in
- RegistrationGuard redirects to /unregistered, if tos/demo survey not submitted or manual IDV not done
- unregistered page autosubmits tos/"demographics survey", requests manual IDV

Alternatively:
- Repurpose the RegistrationGuard to redirect to the homepage instead (if the flag is enabled)
- Autosubmit tos/demo survey from that page if not submitted.
- Somehow also block the user here for manual IDV